microraiden/microraiden/config.py

```diff
@@ -1,80 +1,63 @@
-import json
-import os
 from eth_utils import denoms
-from collections import namedtuple
-
-API_PATH = "/api/1"
-GAS_LIMIT = 130000
-
-# Plain old transaction, for lack of a better term.
-POT_GAS_LIMIT = 21000
-GAS_PRICE = 20 * denoms.gwei
-
-NETWORK_NAMES = {
-    1: 'mainnet',
-    2: 'morden',
-    3: 'ropsten',
-    4: 'rinkeby',
-    30: 'rootstock-main',
-    31: 'rootstock-test',
-    42: 'kovan',
-    61: 'etc-main',
-    62: 'etc-test',
-    1337: 'geth'
+from collections import namedtuple, OrderedDict
+from functools import partial
+
+
+# these are default values for network config
+network_config_defaults = OrderedDict(
+    (('channel_manager_address', None),
+     ('start_sync_block', 0),
+     ('gas_price', 20 * denoms.gwei),
+     ('gas_limit', 130000),
+     # pot = plain old transaction, for lack of better term
+     ('pot_gas_limit', 21000))
+)
+# create network config type that supports defaults
+NetworkConfig = partial(
+    namedtuple(
+        'NetworkConfig',
+        network_config_defaults
+    ),
+    **network_config_defaults
+)
+
+# network-specific configuration
+NETWORK_CONFIG_DEFAULTS = {
+    1: NetworkConfig(
+        channel_manager_address='0x0',
+        start_sync_block=0,
+        gas_price=50 * denoms.gwei
```

Comment:
50 gwei was needed for Ropsten to run the tests faster. This is too much for main net (20 gwei default is fine)